Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.3 → 2.4.0

[_~~change-log~~, ~~source-code~~_]

:tada: Patched vulnerabilities:

- [CVE-2019-15587](https://nvd.nist.gov/vuln/detail/CVE-2019-15587)
  **Loofah XSS Vulnerability**

  
  URL: https://github.com/flavorjones/loofah/issues/171

  In the Loofah gem, through v2.3.0, unsanitized JavaScript may occur in sanitized output when a crafted SVG element is republished.






<details>
<summary>Commits</summary>

A change of **56** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.2.3...v2.4.0).

These are the first 10 commits:
- (5ab19a3) [scripts to inspect and compare DOMPurify metadata](https://github.com/flavorjones/loofah/commit/5ab19a311c75357c06daf8e53712de0a211ade65)
- (2f8d33c) [reformat `whitelist.rb`](https://github.com/flavorjones/loofah/commit/2f8d33c4ba239aac892e825af7d002ac04a493fc)
- (efb182c) [expand set of allowed CSS functions](https://github.com/flavorjones/loofah/commit/efb182cdbc6244f7bb80bb74a6a16261dea98dae)
- (a1b1da9) [expand allowed protocols to allow `tel:` and `line:`](https://github.com/flavorjones/loofah/commit/a1b1da9ca1ce526b02e18667bddc09f97a3de946)
- (cfa40b9) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/cfa40b914a1561a07a9c02288c36ca6c5b072ffe)
- (d10e630) [failing test for high-precision CSS values](https://github.com/flavorjones/loofah/commit/d10e63065eccdea8e0a9939fe22d3357a6cb1ceb)
- (a56a1ab) [Allow greater precision in values of shorthand css elements](https://github.com/flavorjones/loofah/commit/a56a1ab5fbd413d36ca5af098a627f745e53bc63)
- (ef30cfd) [update CHANGELOG](https://github.com/flavorjones/loofah/commit/ef30cfddc2fa9a9fff2ba9a4953ed25e8eb3e05e)
- (717655a) [Merge branch 'flavorjones-allowlist-changes'](https://github.com/flavorjones/loofah/commit/717655a7333486650e1bcb6966827fc87874d38e)
- (b15ea87) [remove versioneye from readme](https://github.com/flavorjones/loofah/commit/b15ea8748d7fb43cf36ce24f8c9bf2f6c9f42354)


</details>